src/domain/manageCartInfo.ts

```
@@ -0,0 +1,13 @@
+import { getCartItems } from '../services/cartItemServices';
+import { CartItemType } from '../types/data';
+
+export const isCartItem = async (productId: number) => {
+  const data = await getCartItems();
+  return data.some((item: CartItemType) => item.product.id === productId);
+};
+
+export const getCartId = async (productId: number) => {
+  const data = await getCartItems();
+  const cartItem = data.find((item: CartItemType) => item.product.id === productId);
+  return cartItem && cartItem.id;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)